src/bootstrap/test.rs

```diff
@@ -2529,7 +2529,13 @@ impl Step for Bootstrap {
     /// Tests the build system itself.
     fn run(self, builder: &Builder<'_>) {
         let mut check_bootstrap = Command::new(&builder.python());
-        check_bootstrap.arg("bootstrap_test.py").current_dir(builder.src.join("src/bootstrap/"));
+        check_bootstrap
+            .arg("-m")
+            .arg("unittest")
+            .arg("bootstrap_test.py")
```

Comment:
```suggestion
            .args(["-m", "unittest", "bootstrap_test.py"])
```

Just personal preference, maybe the convention is the other way.